### PR TITLE
add cli tests for notifications

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
Add cli tests through occ command.

## Related Issue
#242

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    When the administrator sends following notifications using the occ command
+     | subject       | user  |
+     | Quota updated | user0 |
+    Then the command should have been successful
+    And user "user0" should have 1 notifications
+    And the last notification of user "user0" should match
+      | subject | Quota updated |
+
+  Scenario: administrator should be able to send a notification with subject and message to a group
+    Given these users have been created:
```
Comment (dpakach) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]: May be we should check if all the members in a group are getting the notification or not. Try adding at least one more user to the group and check if both the users have notification or not.

**Comment by dpakach** [comment]
LGTM

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 2]
```diff
@@ -0,0 +1,88 @@
+@cli @notification-app-required
+Feature: send notifications using the occ command
+
+  As an administrator
+  I want to be able to send notifications to the owncloud users
+  So that I can inform them about the necessary things
+
+  Scenario: administrator should be able to send a notification with subject and message to a user
+    Given these users have been created:
```
Comment (skshetry) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]:
```suggestion
    Given user "user0" has been created with default attributes
```

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     | subject       | message                                          | user  |
+     | Quota updated | Congratulations your oC quota has been increased | user0 |
+    Then the command should have been successful
+    And user "user0" should have 1 notification
+    And the last notification of user "user0" should match
+      | subject | Quota updated                                    |
+      | message | Congratulations your oC quota has been increased |
+
+  Scenario: administrator should be able to send a notification with a link
+    Given these users have been created:
```
Comment (skshetry) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]:
```suggestion
    Given user "user0" has been created with default attributes
```

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+     | Quota updated | Congratulations your oC quota has been increased | user0 | https://owncloud.org/blog/about-activities-and-notifications-in-owncloud/ |
+    Then the command should have been successful
+    And user "user0" should have 1 notification
+    And the last notification of user "user0" should match
+      | subject | Quota updated                                                             |
+      | message | Congratulations your oC quota has been increased                          |
+      | link    | https://owncloud.org/blog/about-activities-and-notifications-in-owncloud/ |
+
+  Scenario: administrator should be able to send a notification with only the subject to a user
+    Given these users have been created:
```
Comment (skshetry) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]:
```suggestion
    Given user "user0" has been created with default attributes
```

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    When the administrator sends following notifications using the occ command
+     | subject       | group |
+     | Quota updated | grp1  |
+    Then the command should have been successful
+    And user "user0" should have 1 notification
+    And user "user1" should have 1 notification
+    And user "user2" should have 0 notification
+
+  Scenario: administrator should be able to send a notification with subject and message to a group
+    Given these users have been created:
```
Comment (skshetry) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]:
```suggestion
    Given user "user0" has been created with default attributes
```

File: tests/acceptance/features/cliNotifications/sendNotifications.feature [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+     | subject       | message                                          | group  |
+     | Quota updated | Congratulations your oC quota has been increased | grp1   |
+    Then the command should have been successful
+    And user "user0" should have 1 notification
+    And the last notification of user "user0" should match
+      | subject | Quota updated                                    |
+      | message | Congratulations your oC quota has been increased |
+
+  Scenario: administrator sends more than one notifications to a user
+    Given these users have been created:
```
Comment (skshetry) [inline tests/acceptance/features/cliNotifications/sendNotifications.feature]:
```suggestion
    Given user "user0" has been created with default attributes
```

**Comment by skshetry** [comment]
LGTM except for the above comments.

**Comment by phil-davis** [comment]
Backport `stable10` #264

**Pull request merged**